pkg/ir/unspecified.go

```diff
@@ -103,10 +103,10 @@ func (s *Spec) computeBlockedNIFs() []string {
 	return blockedEndpoints
 }
 
-func (s *Spec) findEndpointInConnections(endPoints []string, epType EndpointType) bool {
+func (s *Spec) findEndpointInConnections(endpoints []string, epType EndpointType) bool {
```

Comment:
Why change endpoints to lower case when you are doing the opposite for other variables?
And if so, why leave epType in camel case?

Reply:
Because "endpoint" is a single word, not two words concatenated. I would not write "end point" even if spaces were allowed. That's also the convention in the three repositories. Does goland complain?

I might replace `epType` with `endpintType`.

Reply:
I mean
![image](https://github.com/np-guard/vpc-network-config-synthesis/assets/5108069/4ef496e1-7c19-4b6a-b721-4f60ed5fe2e9)